### PR TITLE
Use older cos-63 version for scale jobs

### PR DESCRIPTION
**Pull request by shyamjvs**
Ref https://github.com/kubernetes/kubernetes/issues/62456

/cc @wojtek-t

### CODE REVIEW

File: jobs/env/ci-kubernetes-e2e-gce-large-performance.env [hunk 1]
```diff
@@ -9,6 +9,8 @@ ENABLE_BIG_CLUSTER_SUBNETS=true
 LOGROTATE_MAX_SIZE=5G
 
 ### kubernetes-env
+# TODO: Change the cos version back to default once #62456 is fixed.
```
Comment (wojtek-t) [inline jobs/env/ci-kubernetes-e2e-gce-large-performance.env]: TODO(shyamjvs)

File: jobs/env/ci-kubernetes-e2e-gce-scale-performance.env [hunk 2]
```diff
@@ -10,6 +10,8 @@ ENABLE_BIG_CLUSTER_SUBNETS=true
 LOGROTATE_MAX_SIZE=5G
 
 ### kubernetes-env
+# TODO: Change the cos version back to default once #62456 is fixed.
```
Comment (wojtek-t) [inline jobs/env/ci-kubernetes-e2e-gce-scale-performance.env]: TODO(shyamjvs)

File: jobs/env/ci-kubernetes-e2e-gce-large-performance.env (hunk 1 above)
Comment (shyamjvs) [reply jobs/env/ci-kubernetes-e2e-gce-large-performance.env]: Done

File: jobs/env/ci-kubernetes-e2e-gce-scale-performance.env (hunk 2 above)
Comment (shyamjvs) [reply jobs/env/ci-kubernetes-e2e-gce-scale-performance.env]: Done

**Pull request merged**